tests/helics/application_api/ValueFederateAdditionalTests.cpp

```diff
@@ -691,14 +693,14 @@ TEST(valuefederate, toml_file_loadb)
 
     EXPECT_EQ(vFed.getInputCount(), 3);
     EXPECT_EQ(vFed.getPublicationCount(), 2);
-    auto& id = vFed.getPublication("primary");
+    auto& pubid = vFed.getPublication("primary");
 
-    EXPECT_EQ(id.getName(), "valueFed_toml/pub2");
+    EXPECT_EQ(pubid.getName(), "valueFed_toml/pub2");
 
-    auto& id2 = vFed.getPublication("pub1");
-    EXPECT_EQ(id2.getUnits(), "m");
-    EXPECT_EQ(id2.getTag("description"), "a test publication");
-    EXPECT_EQ(std::stod(id2.getTag("period")), 0.5);
+    auto& pubid2 = vFed.getPublication("pub1");
```

Comment:
A bit strange how `pubid2` maps to a publication named `pub1`, and `pubid` is one named `valuedFed_toml/pub2`.